### PR TITLE
Added About Us page

### LINKED ISSUE #194: Create About Us Page
## Description
As a contributor,
I need create an about us page (which can be separate from home page or on the home page itself),
so that users can get information about Systers.

## Mocks
screen shot 2018-05-31 at 10 54 24 pm
The about us info which also includes the map as shown below:
screen shot 2018-05-31 at 10 54 01 pm

### Update [Required]
- [x] about.component.html
- [ ] about.component.scss
- [ ] about.component.ts

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
6 hours 

@ancyp @Janiceilene Can you please assign me this issue to work upon? :)

### PR DESCRIPTION
**Pull request by BaaniLeen**
### Description
Added About Us page.

Fixes #194 

### Type of Change:

- Code
  
**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Mockup : screen shot 2018-06-02 at 10 46 35 pm
Mockup for Mobile View : 
screen shot 2018-06-02 at 11 51 15 pm

### Checklist:
**Delete irrelevant options.**

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
  

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings 
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by Janiceilene** [comment]
Where are the numbers for the communities and connected people coming from if the map is just a static image? Is that information also going to be static?

**Comment by divyanshu-rawat** [comment]
Map not visible things get worse when viewed in Mobile.
screen shot 2018-06-03 at 11 41 22 pm
screen shot 2018-06-03 at 11 41 31 pm

**Comment by BaaniLeen** [comment]
@divyanshu-rawat It is due to the nav bar and the footer not being responsive. I have created that as a separate issue( #201 ) and am working on it.
If you remove the footer in the app.component.html file, it should be mobile responsive, as given in my mockup screenshots above. :)

**Comment by ExactlyMay** [comment]
@BaaniLeen Yes, just use that image as a placeholder. Once we get another one from marketing or if I create a simple one, we'll replace it.

**Comment by BaaniLeen** [comment]
Sure @ExactlyMay :)

**Comment by BaaniLeen** [comment]
@divyanshu-rawat @ancyp @Janiceilene  It would be great if you could review the PR once again :)
@divyanshu-rawat I have sent a separate PR for Issue #201 :)

**Comment by ancyp** [comment]
Update time for PR.
It is better to not have the text captured as a image. Here the text next to map is responsive https://community.anitab.org/.
The font on this page does not match the one on contact us page. 
Please make sure to be consistent in fonts on all pages.
The font you have on this page looks nicer. So may be change it on contact us page?

File: src/app/about/about.component.html [hunk 1]
```diff
@@ -1 +1,15 @@
-<h1>ABOUT PAGE</h1>
+<div class="container-fluid mt-5">
+    <div class=" container box logos mx-auto d-block">
```
Comment (ancyp) [inline src/app/about/about.component.html]: space

File: src/app/projects/projects.component.html [hunk 2]
```diff
@@ -1 +1,65 @@
-<h1>PROJECTS PAGE</h1>
+<div class="container-fluid mt-5">
+	<div class=" container box logos mx-auto d-block">
```
Comment (ancyp) [inline src/app/projects/projects.component.html]: space

File: src/app/projects/projects.component.html [hunk 3]
```diff
@@ -1 +1,65 @@
-<h1>PROJECTS PAGE</h1>
```
Comment (ancyp) [inline src/app/projects/projects.component.html]: Why is this file here ?
Aren't you creating new brach from stable master?

Comment (BaaniLeen) [reply src/app/projects/projects.component.html]: @ancyp Currently the Project page has not been merged into the main gsoc18-code branch. That is why perhaps when I made a new branch from it, this default page showed up. :)

**Comment by BaaniLeen** [comment]
@ancyp I have taken care of your suggestions :)
Thanks! :)
Can you review the PR again :)

**Comment by BaaniLeen** [comment]
@Janiceilene I was searching about it and found out that there are some default breakpoints defined which allow a certain screen in the Desktop view to look a certain way in the mobile view. I did try to change that, but I could not do it currently to change that and put the image and the footer text in the same line in mobile view.
Maybe we can look this up later, in the end while improving the UI part? :)
@divyanshu-rawat Can you review the PR? :)

**Comment by BaaniLeen** [comment]
@Janiceilene @divyanshu-rawat Can you please review this PR? It needs a second review by a mentor to be merged. :)

**Comment by ancyp** [comment]
@BaaniLeen Yesterday we discussed about having the text below the image for the map. I hope you are doing that?

**Comment by BaaniLeen** [comment]
@ancyp Sure. I will be doing that. Currently working on the Final Coding Phase 1 Blog :)
